src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -180,7 +179,6 @@ def _accept_cleared_offers(self, clearing_rate, clearing_energy, accepted_bids):
                 break
             elif (math.floor(offer.price / offer.energy)) <= clearing_rate and \
                     (clearing_energy - cumulative_traded_offers) >= offer.energy:
-                # TODO: Used the clearing_rate as the original_trade_rate for the offers, because
```

Comment:
This TODO should not be deleted IMO

Reply:
Oh yes! This was not on purpose. I had my own little TODO there and deleted both. Will re-add again.